oarepo_vocabularies/ext.py

```diff
@@ -33,6 +33,9 @@ def init_config(self, app):
         app.config.setdefault(
             "VOCABULARIES_FACET_CACHE_TTL", config.VOCABULARIES_FACET_CACHE_TTL
         )
+        app.config.setdefault(
```

Comment:
Is there a reason to use `INVENIO_VOCABULARY_` prefix here and not the `OAREPO_VOCABULARIES`, which gets  auto-loaded on line 23 above?